map/extrapolation/extrapolator.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "platform/location.hpp"
+
+#include "base/thread.hpp"
+
+#include <cstdint>
+#include <functional>
+#include <limits>
+#include <mutex>
+#include <thread>
+
+namespace position_extrapolator
+{
+/// \brief Returns extrapolated position after |point2| in |timeAfterPoint2Ms|.
+/// \note This function is assumed that between |point1| and |point2| passed one second.
```

Comment:
s/is assumed/assumes/

Reply:
Why assume this? Can't you get this information from |point1| and |point2|?

Reply:
It's an outdated comment. Removed.